apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -182,6 +183,32 @@ int verify_file(char *path, int bufsize)
 			}
 		}
 	}
+	
+	if (i != TEST_WRITECOUNT) {
+		/* truncate current file to include valid data only */
+		int filesize = i * bufsize;
+		lseek(fd, 0, SEEK_SET);
+		stat(path, &st);
+
+		ret = ftruncate(fd, filesize);
+		if (ret != OK) {
+			printf("ftruncate(shrink)error filename : %s target size : %d errno : %d\n", path, filesize, errno);
+			close(fd);
+			return ERROR;
+		}
+
+		if (stat(path, &st) == OK) {
+			if (st.st_size != filesize) {
+				printf("size is wrong. filename : %s target size : %d file size : %d\n", path, filesize, st.st_size);
+				return ERROR;
```

Comment:
I think line 203 and line 208 need close(fd).

Reply:
yes. I will fix it thanks.